README.md

```diff
@@ -32,6 +33,7 @@ Integrations are chat platforms that Kassy integrates into. The currently availa
 		- [Facebook Integration](doc/integrations/Facebook.md)
 		- [Slack Integration](doc/integrations/Slack.md)
 		- [Skype Integration](doc/integrations/Skype.md)
+                - [Telegram Integration](doc/integrations/Telegram.md)
```

Comment:
Totally odd because I have my editor set to use space, not tabs! I'll double-check. Thanks for the catch. :-)